app/models/long_problem.rb

```diff
@@ -42,6 +42,10 @@ class LongProblem < ActiveRecord::Base
   has_many :submission_pages, through: :long_submissions
   has_many :temporary_markings, through: :long_submissions
 
+  attr_accessor :max_score
```

Comment:
I don't think you need this block anymore. When you added the column to Rails it should auto detect it.

See if you remove this block will it work?